addOns/httpsInfo/src/main/java/org/zaproxy/zap/extension/httpsinfo/ExtensionHttpsInfo.java

```diff
@@ -19,22 +19,17 @@
  */
 package org.zaproxy.zap.extension.httpsinfo;
 
-import java.awt.CardLayout;
-import java.awt.Component;
+import java.awt.*;
```

Comment:
This should be reverted, you might need to configure the IDE to not automatically do this change.

Reply:
Okay I will correct these mistakes soon.